app/views/floating_ip/new.html.haml

```diff
@@ -1,4 +1,8 @@
-%form#form_div{:name => "angularForm", 'ng-controller' => "floatingIpFormController"}
+%form#form_div{:name => "angularForm",
+'ng-controller' => "floatingIpFormController as vm",
```

Comment:
also, please remember to align these lines..

`:name` and `'ng-controller'` should start on the same column, and ideally so should the arrows.